packages/blade/src/components/BaseHeaderFooter/BaseHeader.tsx

```diff
@@ -229,7 +229,7 @@ const _BaseHeader = ({
                 flexDirection="row"
               >
                 {title ? (
-                  <Text size="large" weight="semibold" type="normal">
+                  <Text size="large" weight="semibold" color="surface.text.gray.normal">
```

Comment:
I hope you have verified the `weight` with the new mapping of typography?

Reply:
Yup. Its semibold (600) on figma as well